io.gitlab.gregorni.ASCIIImages.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id" : "io.gitlab.gregorni.ASCIIImages",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "asciiimages",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	{
+	    "name": "blueprint-compiler",
+	    "buildsystem": "meson",
+            "cleanup" : [ "*" ],
+	    "sources": [
+	    	{
+		    "type": "git",
+		    "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+		    "tag": "v0.6.0"
+	    	}
+	     ]
+	},
+        {
+            "name" : "jp2a",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/Talinx/jp2a.git",
+                    "commit" : "988ec7c519ed4b439c13358726559e53ebbdaf3c"
+                },
+                {
+                    "type" : "shell",
+                    "commands": [
+                        "autoreconf -vfi"
+                    ]
+                }
+            ]
+        },
+        {
+            "name" : "asciiimages",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.com/gregorni/asciiimages",
+                    "tag" : "1.0.0",
```

Comment:
```suggestion
```

commit no longer match tag.

Reply:
I'll make a new release, 1.0.1, and specify that.

Reply:
Is it possible to specify a repo without a tag, but with a commit?

Reply:
my suggestion is 100% valid.

commit is a requirement unless tag alone, which is fine. tag + commit protect against retag and is better.

Reply:
Oh. Sorry I didn't apply your suggestion. Anyway, I've just created a new tag now, and specified that along with the latest commit, and it works. Thank you nevertheless.